ding/framework/middleware/collector.py

```diff
@@ -42,6 +59,17 @@ def __call__(self, ctx: "OnlineRLContext") -> None:
 
 
 class EpisodeCollector:
+    """
```

Comment:
move to `__init__` method, class comment should contain the whole overview and interface method list